### PR TITLE
created CovidTest class

### PR DESCRIPTION
**Pull request by anahittum**
The COVID test solution

### CODE REVIEW

File: src/homeworks/week3/covidtest/User.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    public String getCode() {
+        return code;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public static void print(User user) {
```
Comment (gevorgyanarman) [inline src/homeworks/week3/covidtest/User.java:21]: please add non static print method too

File: src/homeworks/week3/covidtest/CovidTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    public User getUser() {
+        return user;
+    }
+
+    public boolean isPositive() {
+        return positive;
+    }
+
+    public static void printCovidTest(CovidTest covidTest) {
```
Comment (gevorgyanarman) [inline src/homeworks/week3/covidtest/CovidTest.java:21]: please add non static print method too

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

File: src/homeworks/week3/covidtest/Main.java [hunk 3]
```diff
@@ -11,5 +11,7 @@ public static void main(String[] args) {
         printCovidTest(covidTest);
         printCovidTest(covidTest2);
         printCovidTest(covidTest3);
+
+        covidTest.printCovidTestNonStatic(covidTest);
```
Comment (anahittum) [inline src/homeworks/week3/covidtest/Main.java:15]: @gevorgyanarman does this make sense at all?

File: src/homeworks/week3/covidtest/CovidTest.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        System.out.print(covidTest.isPositive());
+        System.out.print(" ");
+        User.print(covidTest.getUser());
+    }
+
+    public void printCovidTestNonStatic(CovidTest covidTest) {
```
Comment (gevorgyanarman) [inline src/homeworks/week3/covidtest/CovidTest.java:27]: you do not need to pass a covidTest as an argument.

File: src/homeworks/week3/covidtest/Main.java (hunk 3 above)
Comment (gevorgyanarman) [reply src/homeworks/week3/covidtest/Main.java:15]: yeah, it does. here you shouldn't pass an object as a method argument cuz in class non static method you have it

File: src/homeworks/week3/covidtest/User.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        System.out.println(user.getCode() + " " + user.getName());
+    }
+
+    public void printNonStatic(User user) {
```
Comment (gevorgyanarman) [inline src/homeworks/week3/covidtest/User.java:25]: same here

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

File: src/homeworks/week3/covidtest/Main.java (hunk 3 above)
Comment (anahittum) [reply src/homeworks/week3/covidtest/Main.java:15]: I am not sure I understand your comment. Can you clarify, please?

File: src/homeworks/week3/covidtest/CovidTest.java (hunk 4 above)
Comment (anahittum) [reply src/homeworks/week3/covidtest/CovidTest.java]: Corrected.

**Comment by anahittum** [comment]
@gevorgyanarman I have corrected everything you had commented on. Can this PR be merged?

**Pull request merged**